samples/ComponentExamples/appsettings.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
Assume this file was unintentionally deleted

Reply:
Git is screaming at me by trying to check it in. Says that has secrets

Reply:
Haha, sorry that's my pre-commit hook to ensure we don't accidentally commit a pasted connection string:
![image](https://user-images.githubusercontent.com/2684369/142680635-d6c8fbb9-309b-459f-bd6a-fd2a791b7414.png)

When adding this back in, do `git commit --no-verify` to allow it

Reply:
(though good to know this technique doesn't care about deleting the file)